src/actions/guides/command_line_tasks/custom_tasks.cr

```diff
@@ -7,26 +7,47 @@ class Guides::CommandLineTasks::CustomTasks < GuideAction
 
   def markdown
     <<-MD
-    ## Custom Tasks
+    ## Creating Custom Tasks
 
-    Custom tasks you need will be placed in the `tasks` folder of your application. The three main things to creating a custom task is that your class inherits from `LuckyCli::Task`, it implements a `call` method, and includes a `banner`. Be sure to `require "lucky_cli"` at the top of your new task.
-
-    Lucky will infer the name of the task by using the name of your class. This includes using namespaces. (e.g. `Db::Migrate` becomes `lucky db.migrate`).
+    Custom tasks you need will be placed in the `tasks` folder of your application.
+    The three main things to creating a custom task is
```

Comment:
How about “your custom task must:” and then the list. That way the list items can be shorter, e.g. “include a summary”